src/Styles/trumps/_transition.scss

```diff
@@ -0,0 +1,38 @@
+/* transition: name | duration | easing-function | delay */
+
+.transition {
+  &__withMoveIn {

```

Comment:
Continuas a usar camelCase. Se queres usar esse nome tens de usar hifens: transition__with-move-in
De qualquer forma as classes têm tendencia a evitar preposições. A mesma coisa para a classe abaixo